src/clr-angular/accordion/accordion-panel.html

```diff
@@ -15,7 +15,7 @@
         [attr.aria-controls]="getAccordionContentId(panel.templateId)"
         [attr.aria-expanded]="panel.open"
         [attr.aria-describedby]="getAccordionStatusId(panel.templateId)"
-        [class.clr-accordion-header-has-description]="description.children.length != 0"
+        [class.clr-accordion-header-has-description]="(accordionDescription.changes | async)?.length || accordionDescription.length"
```

Comment:
Unfortunately changes does not emit initial value. This gets changes + initial without creating chocolate errors